app/models/reaction_category.rb

```diff
@@ -0,0 +1,63 @@
+# See also reactions.yml and initializers/load_reaction_category_list.rb
+class ReactionCategory
+  class << self
+    def [](slug)
+      list[slug.to_sym]
+    end
+
+    def all_slugs
+      list.values.map(&:slug)
```

Comment:
We could have a separate method with `list.values`(maybe a more meaningful namge + cache it), I suppose it would make understanding easier.

Reply:
Maybe it's better in [3ecef10a3](https://github.com/forem/forem/blob/3ecef10a3936859d08b14c3a98a6c2665e9a8cfe/app/models/reaction_category.rb) ?